ministryofjustice/analytical-platform-infrastructure
# Home Dir Backups

- Creates S3 bucket for logging
- Creates S3 bucket to save home directory backups from SerjNas
- Creates needed IAM resources for the cluster to save backups

Module to create:
- IAM policy to manage NFS backup bucket
- IAM role to be assumed by K8s cluster
- S3 bucket to store backups

Created Generic logging bucket to capture events

Added Spec tests

**Deployed to Prod** 👍 

## To Test
You must ensure you have your local landing IAM profile setup. See [here](https://docs.aws.amazon.com/cli/latest/userguide/cli-configure-profiles.html)

1. Remove  lines `10` and `17` in https://github.com/ministryofjustice/analytical-platform-infrastructure/blob/master/spec/spec_helper.rb

2. Now run `bundle exec rake spec`
